pkg/gadgets/internal/networktracer/tracer.go

```diff
@@ -274,7 +274,15 @@ func (t *Tracer[Event]) Detach(pid uint32) error {
 	return fmt.Errorf("pid %d is not attached", pid)
 }
 
+// Close closes the tracer
+// TODO: Remove after refactoring
 func (t *Tracer[Event]) Close() {
+	t.Cleanup()
+}
+
+// ---
+
+func (t *Tracer[Event]) Cleanup() {
```

Comment:
Why this change? What's wrong with having a `Close()` method? I think Close() is very well-known in go as the method that performs the cleanup for a resource, so probably we want to keep using that name.